File: src/main/java/com/vaadin/demo/component/login/LoginOverlayFooter.java
package com.vaadin.demo.component.login;

import com.vaadin.demo.DemoExporter; // hidden-source-line
import com.vaadin.flow.component.login.LoginOverlay;
import com.vaadin.flow.component.html.Div;
import com.vaadin.flow.component.html.Paragraph;
import com.vaadin.flow.router.Route;
import com.vaadin.flow.theme.lumo.LumoUtility;

@Route("login-overlay-footer")
public class LoginOverlayFooter extends Div {

    public LoginOverlayFooter() {
        // tag::snippet[]
        LoginOverlay loginOverlay = new LoginOverlay();
        Paragraph text = new Paragraph("Never tell your password to anyone");
        text.addClassName(LumoUtility.TextAlignment.CENTER);
        loginOverlay.getFooter().add(text);
        // end::snippet[]
        add(loginOverlay);
        loginOverlay.setOpened(true);
        // Prevent the example from stealing focus when browsing the
        // documentation
        loginOverlay.getElement().setAttribute("no-autofocus", "");
    }

    public static class Exporter extends DemoExporter<LoginOverlayFooter> { // hidden-source-line
    } // hidden-source-line
}

File: frontend/demo/component/login/login-overlay-footer.ts
import 'Frontend/demo/init'; // hidden-source-line
import '@vaadin/login';
import { html, LitElement } from 'lit';
import { customElement } from 'lit/decorators.js';
import { applyTheme } from 'Frontend/demo/theme';

@customElement('login-overlay-footer')
export class Example extends LitElement {
  protected override createRenderRoot() {
    const root = super.createRenderRoot();
    applyTheme(root);
    return root;
  }

  protected override render() {
    return html`
      <!-- tag::snippet[] -->
      <!-- no-autofocus is used to prevent the example from stealing focus when browsing the documentation -->
      <vaadin-login-overlay opened no-autofocus>
        <p slot="footer" class="text-center">Never tell your password to anyone</p>
      </vaadin-login-overlay>
      <!-- end::snippet[] -->
    `;
  }
}
